service/src/main/java/org/apache/celeborn/server/common/service/config/DynamicConfigServiceFactory.java

```diff
@@ -32,12 +44,13 @@ public static ConfigService getConfigService(CelebornConf celebornConf) throws I
     if (_INSTANCE == null) {
       synchronized (DynamicConfigServiceFactory.class) {
         if (_INSTANCE == null) {
-          String configStoreBackend = celebornConf.dynamicConfigStoreBackend().get();
-          if ("FS".equals(configStoreBackend)) {
-            _INSTANCE = new FsConfigServiceImpl(celebornConf);
-          } else {
-            _INSTANCE = new DbConfigServiceImpl(celebornConf);
-          }
+          String configStoreBackendName = celebornConf.dynamicConfigStoreBackend().get();
+          String configStoreBackendClass =
+              dynamicConfigStoreBackendShortNames.getOrDefault(
```

Comment:
The custom `ConfigService` implementation could not add short name in `dynamicConfigStoreBackendShortNames`.

Reply:
i was sort of following what spark does for shuffle manager. It allows short names for `sort` and `tungesten-sort` but if user is passing custom shuffle manager then they have to pass fully qualified name.